tests/evm.rs

```diff
@@ -255,7 +255,7 @@ fn ethereum_solidity_tests() {
         })
         .sum();
 
-    assert_eq!(errors, 1155);
+    assert_eq!(errors, 1094);
```

Comment:
Why has this changed?

Reply:
Because this change fixes a whole bunch of failures in the eth-solidity-tests-through-sema tests